lib/active_model/serializer/adapter.rb

```diff
@@ -11,6 +12,47 @@ class Adapter
 
       attr_reader :serializer
 
+      # Only the Adapter class has these methods.
+      # None of the sublasses have them.
+      class << ActiveModel::Serializer::Adapter
+        def adapter_map
+          ADAPTER_MAP
+        end
+
+        def register(name, klass)
+          adapter_map.update(name.to_s.underscore => klass)
+        end
+
+        def get(adapter)
```

Comment:
@bf4 this seems almost the same logic of the `adapter` [method](https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model/serializer.rb#L134)
We should probably remove it from there in order to use this new `get` method, right?